digi/templates/includes/projects_section.html

```diff
@@ -2,7 +2,11 @@
 
 <section class="section--projects sub-section--projects">
     <div class="container">
-      <h2 class="text-center">{{ section_title }}</h2>
+      {% if page.parent_theme %}
+      <h2 class="text-center sub-header"><a href="{% pageurl page.parent_theme %}">{{ section_title }}</a></h2>
+      {% else %}
+      <h2 class="text-center sub-header">{{ section_title }}</h2>
```

Comment:
This should be DRY Use if statement only for the link, not for the header element which causes duplications.